lib/levelup.js

```diff
@@ -377,8 +377,8 @@ LevelUP.prototype.createReadStream = function (options) {
   var self = this
   options = extend( {keys: true, values: true}, this.options, options)
 
-  options.keyEncoding   = options.keyEncoding   || options.encoding
```

Comment:
should we add a test for this? `.encoding` was always only meant as an alias for `.valueEncoding`